lib/client/api.go

```diff
@@ -2993,7 +2999,7 @@ func (tc *TeleportClient) PingAndShowMOTD(ctx context.Context) (*webclient.PingR
 
 	pr, err := tc.Ping(ctx)
 	if err != nil {
-		return nil, trace.Wrap(err)
+		return nil, trace.Wrap(err, "Teleport proxy not available at proxy address %s. Confirm address and connectivity.", tc.WebProxyAddr)
```

Comment:
```suggestion
		return nil, trace.Wrap(err, "Teleport proxy not available at %s.", tc.WebProxyAddr)
```

I don't think `Confirm address and connectivity` is necessary.